.gitignore

```diff
@@ -16,3 +16,5 @@ DerivedData
 .idea/
 *.hmap
 *.xccheckout
+.swiftpm
+.swift-version
```

Comment:
Why do we need to ignore here swift-version?

Reply:
These files are autogenerated files. So we don't need to include on git